### PR TITLE
Data input

### PR DESCRIPTION
**Pull request by bh1995**
Python script for the input data

### CODE REVIEW

File: gcn_prot/data/protien_graph.py [hunk 1]
```diff
@@ -31,7 +30,7 @@ def __init__(self, data, nb_nodes=185, task_type="classification", nb_classes=2)
         self.nb_classes = nb_classes
         self.task_type = task_type
 
-        self.ident = np.eye(nb_nodes)
+        self.ident = np.exp(nb_nodes)
```
Comment (carrascomj) [inline gcn_prot/data/protien_graph.py]: Why? Take a look at https://docs.scipy.org/doc/numpy/reference/generated/numpy.eye.html. Here, a identity matrix is produced from the size of the aminoacids to then build the mask (so 1 are substituted with 0 in the diagonal if the atom was missing or not well resoluted by the experimental method).

**Review by carrascomj — CHANGES_REQUESTED** [review]
The file `data_input.py` is missing. Maybe you have forgotten to commit it to the branch (?) :).

File: gcn_prot/data/data_input.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+@author: Bjorn
+"""
+import torch as t
+import torch.nn.functional as f
+
+
+def VCAInput(n_nodes, n_coord, n_feat, mask=None):
+   
+   '''
+    Method returns Pytorch placeholders for graph convolutional networks.
```
Comment (carrascomj) [inline gcn_prot/data/data_input.py]: Yes, in TensorFlow we would need placeholders. However, what we need for Pytorch is a transformation from the real ( `[v, c, m, y]` output of `__get_index__` at `gcn_prot/data/protein_graph.py`) to the output matrices (features and adjancency) + labels (which is `y`, untouched).

File: gcn_prot/data/data_input.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      #a = tf.add(-a, 1, name=namespace)
+
+   return a
+   
+  
+#%%
+# input_shape for the kinase data set: [344,29,3]
+# V, C, A, M = VCAInput(input_shape[0], input_shape[2], input_shape[1])
+# VCAInput(nb_nodes, nb_coords, nb_features, a_mask=None)
+input_shape = [344,29,3]
```
Comment (carrascomj) [inline gcn_prot/data/data_input.py]: As a said, instead from shapes, we need to start from a real protein vector `[v, c, m, a]`. Just remove the lines outside the functions (when you are ready to merge) and I'll write the required tests for travis.

**Review by carrascomj — CHANGES_REQUESTED** [review]
Can you run black and isort the files?

```bash
pip install black isort
black path_to_data_input.py
isort path_to_data_input.py
```

File: gcn_prot/data/euclidean_distance.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+   c : Rank 3 array defining coordinates of nodes in n-euclidean space
+
+
+   Returns
+   -------
+   a - Rank 3 tensor defining pairwise adjacency matrix of nodes.
+   '''
+   
+   adj_mat = t.ones(())
+   adj_mat = adj_mat.new_empty((len(c), len(c)))
```
Comment (carrascomj) [inline gcn_prot/data/euclidean_distance.py]: You could just use torch.FloatTensor(len(c), len(c))

File: gcn_prot/data/euclidean_distance.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+   for i in range(0, len(c)):
+      node1 = c[i]
+      node1 = t.from_numpy(node1)
+      for j in range(0, len(c)):
+         node2 = c[j]
+         node2 = t.from_numpy(node2)
+         dist = t.dist(node1, node2)
```
Comment (carrascomj) [inline gcn_prot/data/euclidean_distance.py]: you could just put `dist = ((node1-node2)**2).sum().sqrt()` so that you needn't the `.from_numpy()` nuisance but that's fine too.

File: gcn_prot/data/data_input.py [hunk 6]
```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
```
Comment (carrascomj) [inline gcn_prot/data/data_input.py]: We can just remove this file for now.

File: gcn_prot/data/adjacency_matrix.py [hunk 7]
```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
```
Comment (carrascomj) [inline gcn_prot/data/adjacency_matrix.py]: We can also remove this file.

**Review by carrascomj — APPROVED** [review]
